gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -492,8 +494,26 @@ public BasicInputArea(Composite parent, Settings settings, Theme theme,
               withIndents(new GridData(), GROUP_INDENT, 0));
           gpuCountersLabels = new Label[3];
           gpuCountersLabels[1] = createLabel(counterGroup, "Poll Rate:");
-          gpuCountersRate = createSpinner(counterGroup, sGpu.getCounterRate(), 1, 1000);
-          gpuCountersLabels[2] = createLabel(counterGroup, "ms");
+
+          long minSamplingPeriod = NANOSECONDS.toMicros(gpuCaps.getGpuCounterDescriptor().getMinSamplingPeriodNs());
+          long maxSamplingPeriod = gpuCaps.getGpuCounterDescriptor().getMaxSamplingPeriodNs();
```

Comment:
Shouldn't you also have NANOSECONDS.toMicros(...) around ...getMaxSamplingPeriod**Ns**() ?